docs/API_Reference.md

```diff
@@ -638,3 +643,259 @@ ehsm-core enclave will decrypt user-supplied ciphertextblob with specified CMK t
   ```
   *(return to the [Cryptographic Functionalities APIs](#eHSM-REST-API-Reference).)*
 ---
+
+## ListKey
+Query all CMKs of the current appid.
+
+- **Rest API format:**
+
+  POST <ehsm_srv_address>/ehsm?Action=ListKey
+
+
+- **Response Data:**
+
+  | Name | Type | Reference Value | Description |
+  |:-----------|:-----------|:-----------|:-----------|
+  | code | int | 200  | The result of the method call, 200 is success, others are fail. |
+  | message | String  | "success" | The description of result. |
+  | list | JsonArray | [<br/>&emsp;{<br/>&emsp;&emsp;"keyid": "2de54366-30f9-6829-8391-c4eba&lowast;&lowast;&lowast;", <br/>&emsp;&emsp;"creationDate": 1645152619225, <br/>&emsp;&emsp;"expireTime": 1676688619225, <br/>&emsp;&emsp;"alias": "alias_name", <br/>&emsp;&emsp;"keyspec": "EH_RSA_3072", <br/>&emsp;&emsp;"keyState": 1<br/>&emsp;}<br/>] | <br/>The information of each CMK is a JsonObject.<br/>&emsp;keyid: ID of CMK<br/>&emsp;creationDate: Creation date of CMK<br/>&emsp;expireTime: Expire time of CMK<br/>&emsp;alias: Alias of CMK<br/>&emsp;keyspec: keyspec of CMK<br/>&emsp;keyState: State of CMK, 1 (Enable) &#124; 0 (Disable). |
+
+- **Example**
+  - Request sample in python
+  ```python
+    params = OrderedDict()
+    params["appid"] = appid
+    params["timestamp"] = str(int(time.time() * 1000))
+
+    sign_string = urllib.parse.unquote(urllib.parse.urlencode(params))
+    sign = str(base64.b64encode(hmac.new(appkey.encode('utf-8'), sign_string.encode('utf-8'), digestmod=sha256).digest()),'utf-8').upper()
+
+    params["sign"] = sign
+    
+    requests.post(url="<ehsm_srv_address>/ehsm?Action=ListKey", data=json.dumps(params), headers=headers)
+  ```
+
+  - Response data
+  ```python
+    Response= {
+      "code": 200,
+      "message": "success!",
+      "result": {
+         "list": [
+             {
+                "keyid": "2de54366-30f9-6829-8391-c4eba***", 
+                "creationDate": 1645152619225, 
+                "expireTime": 1676688619225, 
+                "alias": "alias_name", 
+                "keyspec": "EH_RSA_3072", 
+                "keyState": 1
+             }
+         ]
+      }
+    }
+  ```
+  *(return to the [Cryptographic Functionalities APIs](#eHSM-REST-API-Reference).)*
+---
+
+## DeleteKey
+Delete a CMK.
+
```

Comment:
Delete a specific CMK generated by the current account.

Reply:
It will be fix